Host fails to upload debug info.

Running `flynn-host upload-debug-info` produces:

```
23:33:32.334497 upload-debug-info.go:48: error adding tmp-flynn-host.log: open /tmp/flynn-host.log: no such file or directory
```

Host is Ubuntu 14.04 LTS, 3.13.0-46-generic, x86_64.

--- a/host/cli/upload-debug-info.go
+++ b/host/cli/upload-debug-info.go
@@ -44,7 +44,7 @@ func runUploadDebugInfo() error {
 	}
 
 	for name, filepath := range logs {
-		if err := gist.AddLocalFile(name, filepath); err != nil {
+		if err := gist.AddLocalFile(name, filepath); err != nil && !os.IsNotExist(err) {
 			log.Printf("error adding %s: %s", name, err)
 		}
 	}

